### PR TITLE
Adding survey question template

### PR DESCRIPTION
**Pull request by trickwide**
Adding survey template, questions and answers are pulled from JSON file (resides in directory backend/src/static), created necessary routing changes. Currently there is 'Kysymys' option on navigation bar to have ease of testing the survey template in development, subject to change. User can navigate between previous and next question.

Need merge to main for others to see how the current iteration of survey questions work.

### CODE REVIEW

File: backend/src/routes.py [hunk 1]
```diff
@@ -13,3 +16,33 @@ def apitest():
...
 
     return res
+
+
+def load_json():
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    questions_file_path = os.path.join(
+        dir_path, 'static', 'questions.json')
+    try:
+        with open(questions_file_path, 'r') as file:
```
Comment (PaulusParssinen) [inline backend/src/routes.py:26]: Specify encoding for the file read, pylint is crying about this.

```suggestion
        with open(questions_file_path, 'r', encoding='utf-8') as file:
```

File: frontend/src/components/Question.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    const [selectedOption, setSelectedOption] = useState(null)
+    const [questionData, setQuestionData] = useState(null)
+    const [totalQuestions, setTotalQuestions] = useState(null)
+
+    useEffect(() => {
+        const fetchTotalQuestions = async () => {
+            try {
+                const response = await fetch(
+                    `http://127.0.0.1:5000/api/question`
```
Comment (PaulusParssinen) [inline frontend/src/components/Question.jsx:39]: Vite will proxy the request to the backend and in staging/production the request will be handled by the Route object and forwarded to the backend.

```suggestion
                    `/api/question`
```

File: frontend/src/components/Question.jsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        }
+
+        fetchTotalQuestions()
+    }, [])
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await fetch(
+                    `http://127.0.0.1:5000/api/question/${questionId}`
```
Comment (PaulusParssinen) [inline frontend/src/components/Question.jsx:59]: Same here.
```suggestion
                    `/api/question/${questionId}`
```

**Review by PaulusParssinen — CHANGES_REQUESTED** [review]
Couple small changes needed. You can also suppress the pylinter errors, like the one thrown for ``id``. You can add ``good-names=id`` to the `.pylintrc` for example. I think the pylint should be able to reorder the imports to its liking too(?)

Otherwise :shipit:

**Review by Desipeli — APPROVED** [review]
LGTM

**Review by PaulusParssinen — APPROVED** [review]

**Pull request merged**